hawkbit-runtime/hawkbit-update-server/src/main/java/org/eclipse/hawkbit/app/MyUI.java

```diff
@@ -36,7 +36,7 @@
  *
  */
 @SpringUI
-@Push(value = PushMode.AUTOMATIC, transport = Transport.WEBSOCKET)
+@Push(value = PushMode.AUTOMATIC, transport = Transport.WEBSOCKET_XHR)
```

Comment:
A comment explaining why we use "WEBSOCKET_XHR" (as opposed to plain "WEBSOCKET") could be helpful

Reply:
done